utils/index.js

```diff
@@ -0,0 +1,105 @@
+const _ = require('underscore.string');
+const conflict = require('gulp-conflict');
+const fs = require('fs');
+const gulp = require('gulp');
+const inquirer = require('inquirer');
+const install = require('gulp-install');
+const meow = require('meow');
+const mergeStream = require('merge-stream');
+const path = require('path');
+const pkgDir = require('pkg-dir');
+const rename = require('gulp-rename');
+const template = require('gulp-template');
+
+const INTERPOLATION_PATTERN = /<%=([\s\S]+?)%>/g;
+const ROOT_DIR = pkgDir.sync(process.cwd());
+
+/**
+ * Given an options object, return a slush generator function.
+ *
+ * @param {Object} opts
+ * @return {Function}
+ */
+const createGenerator = (opts) => {
+  const {
+    callback,
+    prompts,
+    mapArgsToAnswers = (args) => args,
+    onError = (err, done) => done(err),
+  } = opts;
+
+  return (args = [], flags = {}) => (done) => {
+    const proc = args && args.length
+      ? Promise.resolve(mapArgsToAnswers(args, flags))
+      : inquirer.prompt(prompts);
+
+    return proc
+      .then((answers) => callback(answers, done))
+      .catch((err) => onError(err, done));
+  };
+};
+
+/**
+ * Replace placeholder values using the data provided.
+ *
+ * @param {Object} data
+ */
+const interpolate = (data) => template(data, { interpolate: INTERPOLATION_PATTERN });
+
+/**
+ * Given a pascal case string, convert it to kebab case.
+ *
+ * @param {string} str
+ * @return {string}
+ */
+const pascalToKebab = (str) => {
```

Comment:
Seems like something someone else would have already written. Don't we use a library in core that does this?

Reply:
@jmbtutor This looks promising/is well maintained: https://github.com/blakeembrey/change-case#readme.

Reply:
Looks like a decent library. Let's go with that.

Reply:
Looks like a decent library. Let's go with that. If you want to minimize dependencies, you can also just import the underlying library: https://github.com/blakeembrey/pascal-case

Reply:
Bespoke `pascalToKebab()` has been replaced with `case` lib. Resolving.

RE: `pascal-case`: It looks like that package transforms strings from ____ *to* pascal case. In this case, we're looking for a transformation from pascal to kebab. Based on that, I opted to import the entire `case` library instead.